.github/workflows/python-app.yml

```diff
@@ -0,0 +1,106 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  linux:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Add release key
+      run: echo ${{ secrets.RELEASE_KEY }} > keys/build_key.txt
```

Comment:
Builds started by the `pull_request` target don't have access to repository secrets (in order to prevent secrets from being stolen). Having a blank build key shouldn't cause any problems though.